src/ProcessManagerBundle/Resources/public/pimcore/js/processes.js

```diff
@@ -207,6 +208,21 @@ pimcore.plugin.processmanager.processes = Class.create({
                         }
                     ]
                 },
+                {
+                    text : t('processmanager_artifact_download'),
+                    xtype:'actioncolumn',
+                    width:50,
+                    items: [
+                        {
```

Comment:
How to skip drawing an icon here if there's no artifact attached?

Reply:
@dpfaffenbauer if you can help here, I think this is ready to merge.

Reply:
Easy solution: iconCls can be a function as well, and you could just return an empty string. You also need to check in the handler function if there is anything in there. 

Good and harder solution: https://github.com/w-vision/ImportDefinitions/blob/master/src/ImportDefinitionsBundle/Resources/public/pimcore/js/definition/item.js#L610